### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by supunharsha**
Added unite tests

### CODE REVIEW

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthUserStoreConfigListenerImplTest.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ruwanta) [inline components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthUserStoreConfigListenerImplTest.java]: 2017

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthTenantMgtListenerImplTest.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (ruwanta) [inline components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthTenantMgtListenerImplTest.java]: 2017

Comment (supunharsha) [reply components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthTenantMgtListenerImplTest.java]: done

File: components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthUserStoreConfigListenerImplTest.java (hunk 1 above)
Comment (supunharsha) [reply components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth2/internal/OAuthUserStoreConfigListenerImplTest.java]: done

**Pull request merged**